### ISSUE TITLE
Instructions to download for windows systems

### ISSUE THREAD

**Issue by burstWizard** [KaiDMML/FakeNewsNet#57]

**Comment by kkhanh4github** [comment]
@burstWizard did you solve the issue?
when I execute the command **nohup python -m resource_server.app &> keys_server.out&** in code directory it gives me this issue **_& was unexpected at this time._**
Did you face this issue?

**Comment by burstWizard** [comment]
Yeah that's exactly the issue I faced. I didn't actually need twitter data for my project, just the articles, so I didn't end up running the resource server. I ran the main file using `python main.py`. I think the rest is actually implied for windows. 

However, I did try to get the resource server running at some point, and I think I used python -m resource_server. I ran into some issues since I ran the file from within its own directory, so it couldn't access the keys file properly. I had to reorder the file structure a bit and it eventually worked, though I didn't need it.
